firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/core/UserMetadata.java

```diff
@@ -37,6 +38,7 @@ public void setUserId(String identifier) {
     userId = sanitizeAttribute(identifier);
   }
 
+  @NonNull
   public Map<String, String> getCustomKeys() {
     return Collections.unmodifiableMap(attributes);
```

Comment:
Not sure it matters, but just a note here that unmodifiable maps can still change, just not by the user of the returned value.

Reply:
:+1: Discussed this offline - the `ConcurrentHashMap` implementation guarantees this is safe, but we should definitely revisit it later.